enterprise-suite/templates/es-grafana-deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
             value: "false"
           - name: GF_ANALYTICS_CHECK_FOR_UPDATES
             value: "false"
+          {{- range $key, $val := .Values.esGrafanaConfigs }}
+          - name: {{ $key }}
+            value: {{ $val | quote }}
+          {{- end }}
```

Comment:
Are there any values above that should be moved so they are configurable as well?

Reply:
Having this loop at the end allows a user to override any of the settings above.  (Just confirmed with a test.)  I actually wondered if there were any we _didn't_ want to allow the user to modify.

I figure they're free to break things if they want though...

Reply:
Yes, possibly. It's the most flexible approach in case a customer wants to change something.

Unfortunately this is sort of the nature of helm - to make it customizable, you eventually end up putting everything in values.yaml.